deploy/Dockerfile

```diff
@@ -45,4 +45,7 @@ RUN cp /database-stream-processor/deploy/prometheus_datasource.yaml /etc/grafana
    && cp /database-stream-processor/deploy/grafana_dashboard_provision.yaml /etc/grafana/provisioning/dashboards/ \
    && cp /database-stream-processor/deploy/grafana_dashboard.json /etc/grafana/provisioning/dashboards/
 
-CMD service postgresql start && bash
+CMD service postgresql start && \
+    mkdir /working-dir && \
```

Comment:
I don't think this is necessary, the manager will create the dir if it doesn't exist